bricks.js

```diff
@@ -0,0 +1,18 @@
+function bordermouse(id) {
```

Comment:
Consider factoring out these `style.fill` settings into a CSS class. That way if you decide to change the colors, you can do it in your style file, not the javascript file. So you'd have `id.className = "clicked"`, and in a `bricks.css` file, `.clicked { fill: "black"; }`. Also, rather than handle both mouseover and mouseout events, you can use the CSS `:hover` selector. [CSS Selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors). Don't forget to include the CSS file in the HTML file!